src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const NEW_STATE = { ...state };
```

Comment:
The object 'NEW_STATE' should be created inside the loop to avoid mutating the same object across different actions. Otherwise, all actions will mutate this single object instead of creating a separate state for each action.